WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -441,7 +440,7 @@ protected void onCreate(Bundle savedInstanceState) {
         FragmentManager fragmentManager = getSupportFragmentManager();
         Bundle extras = getIntent().getExtras();
         String action = getIntent().getAction();
-        boolean isRestarting = !RestartEditorOptions.NO_RESTART.name().equals(extras.getString(EXTRA_RESTART_EDITOR));
+        boolean isRestarting = checkToRestart(getIntent());
```

Comment:
see https://github.com/wordpress-mobile/WordPress-Android/pull/12081 (unrelated to Stories, but needed to work well here)